orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -134,13 +147,22 @@ public void init() throws DBException {
       }
     }
 
-    System.err.println("OrientDB connection created with " + url);
+    log.info("OrientDB connection created with " + url);
     dictionary = db.getMetadata().getIndexManager().getDictionary();
     if (!db.getMetadata().getSchema().existsClass(CLASS)) {
       db.getMetadata().getSchema().createClass(CLASS);
     }
 
-    db.declareIntent(new OIntentMassiveInsert());
+    if (intent.equals(ORIENTDB_MASSIVEINSERT)) {
```

Comment:
Should these be set automatically per type of action? Like insert should be massiveinsert, read should be massiveread, etc? Or is it assumed that these settings will be set when someone runs a workload?